apps/web/src/actions/geo.ts

```diff
@@ -0,0 +1,21 @@
+import { cache } from "react";
+
+export interface OpenStreetMapProps {
+    lat: number;
+    lon: number;
+    display_name: string;
+}
+
+export const lookupGeoLocation = cache(async (address: string) => {
```

Comment:
cache funktioniert bei actions nicht, sondern nur bei queries. Müsstest du also rausnehmen
```suggestion
export const lookupGeoLocation = async (address: string) => {
```